docs/README.md

```diff
@@ -25,23 +25,23 @@ Install these dependencies:
 
 ### MacOS
 
-1. Install [doxygen](http://www.stack.nl/~dimitri/doxygen/download.html).
-2. Make doxygen available on the command line.
-  1. Locate your doxygen executable.  One possible location is
-     `/Applications/Doxygen.app/Contents/Resources/doxygen`
-  2. Make that executable available in your path.  For example you could add a
-     symbolic link from `/usr/local/bin` to the doxygen executable:
-     `$ ln -s /Applications/Doxygen.app/Contents/Resources/doxygen /usr/local/bin/doxygen`
-3. Install [Boost](http://www.boost.org/users/download/)
+1. Install doxygen:
+  * Use homebrew to install: `brew install doxygen`.  The executable will be
+    installed in `/usr/local/bin` which is already in your path.
+  * Alternatively, install from here: [doxygen](http://www.stack.nl/~dimitri/doxygen/download.html).
+    You'll then need to make doxygen available to your command line.  You can
+    do this by adding a symbolic link from `/usr/local/bin` to the doxygen
+    executable.  For example, `$ ln -s /Applications/Doxygen.app/Contents/Resources/doxygen /usr/local/bin/doxygen`
+2. Install [Boost](http://www.boost.org/users/download/)
```

Comment:
I use the homebrew boost for building rippled and have no problems (it links and runs fine). There are a few options in the brew build: the only one worth considering (I think) is:  `--c++11`.  

All that said, the brewed boost seems to work fine for rippled, but I could NOT for the life of me get the `b2` that it installs to work with our doc toolchain. For that, I have to use a local from source build of boost. If someone can figure out how to make the `b2` in brew work with our docs build, then I'd be fine with recommending the brewed boost.